src/goSignature.ts

```diff
@@ -94,6 +94,12 @@ export class GoSignatureHelpProvider implements SignatureHelpProvider {
 		for (let lineNr = position.line; lineNr >= 0 && maxLookupLines >= 0; lineNr-- , maxLookupLines--) {
 
 			const line = document.lineAt(lineNr);
+
+			// Stop processing if the line is a comment
+			if (line.text.trim().indexOf('//') === 0) {
+				return null;
```

Comment:
This will only catch the case where the entire line starts with the comment.
What about the below case?
```go
x, err := regexp.Match("abc", data) // I have some comment here,
```

Reply:
That is the most basic case of course. Either way, I'm noticing the bug isn't completely dependent on comments presence so there is still something else in it

Reply:
I decided to reuse the same function made for suggestions logic on this one